lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -198,11 +183,6 @@ def get_minor_ticks(self, numticks=None):
                 obj.set_transform(self.axes.transData)
         return ticks
 
-    @_api.deprecated("3.6")
-    def set_pane_pos(self, xys):
-        """Set pane position."""
-        self._set_pane_pos(xys)
-
     def _set_pane_pos(self, xys):
```

Comment:
You can inline _set_pane_pos to its only call site too (as I did at https://github.com/matplotlib/matplotlib/pull/25373/files#diff-ea1dd4bc7c3b7c0078e69e41e916d25354864e4f2fe31fd71210477f6f9f9660R310; note in particular that the staleness-setting in _set_pane_pos is redundant as we should be in the middle of a draw and thus clearing staleness right now.